envFiles/.env.production

```
@@ -103,3 +103,11 @@ REDIS_MAPPED_PORT=6379
 ############## frontend URL ##############
 
 FRONTEND_URL="http://localhost:4321"
+
+
+##### OTEL and Observability #####
+API_OTEL_ENABLED=true
+API_OTEL_SERVICE_NAME="talawa-api"
+API_OTEL_ENVIRONMENT="production"
+API_OTEL_SAMPLING_RATIO="1"
+API_OTEL_EXPORTER_OTLP_ENDPOINT="http://localhost:4318/v1/traces "#TODO: after integrating Observability tool
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)